build-support/docker/pants_dev/Dockerfile

```diff
@@ -32,6 +32,8 @@ FROM anapsix/alpine-java:8_jdk
 RUN apk update && apk add bash curl gcc git libffi-dev linux-headers musl-dev openssl python python-dev
 
 # Clone the pantsbuild repo and bootstrap Pants in it.
-RUN git clone https://github.com/pantsbuild/pants.git && cd /pants && ./pants
+# The bootstrapping run of ./pants may not return exit code 0 due to unimportant warnings,
+# so we force the exit code to 0, allowing docker build to complete.
+RUN git clone https://github.com/pantsbuild/pants.git && cd /pants && (./pants || :)
```

Comment:
because 
```[tw-mbp-yic source (master)]$ ./pants
No goals specified.
Use `pants goals` to list goals.
Use `pants help` to get help.
[tw-mbp-yic source (master)]$ echo $?
1```